src/components/PageNavigation/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from "react"
+import * as SC from "./styles"
+
+interface IPageNavigationPage {
+  relativePath: string
+  title: string
+}
+
+interface IPageContentProps {
+  next?: IPageNavigationPage
+  previous?: IPageNavigationPage
+}
+
+export const PageNavigation: FC<IPageContentProps> = ({ previous, next }) => {
+  return (
+    <SC.PageNavigationContent>
+      {previous ? (
+        <SC.PageNavigationPageContent>
```

Comment:
kind of nitpicking on naming here, but I wouldn't prefix all of the styles with `PageNavigation`. `SC.Text`, `SC.Link`, `SC.Title` would be fine (and easier to read)

Reply:
I too thought it's bit redundant. Thanks for suggesting names for it.